packages/client/hmi-client/src/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -208,16 +234,89 @@ const inputChangeHandler = async () => {
 	}
 };
 
-const saveNewModel = async () => {
-	if (!model.value || !newModelName.value) return;
-	model.value.header.name = newModelName.value;
+const saveNewModel = async (modelName: string, options: Record<string, boolean>) => {
```

Comment:
Since you know exactly what may be passed in, please avoid generic`Record<...>`, just create a local interface of it or something, this makes things a bit clearer.

```
interface SaveOptions {
  addToProject?: boolean;
  addToPortXYZ?: boolean;
}
```

Reply:
done